cmd/koord-manager/main.go

```diff
@@ -78,10 +80,18 @@ func init() {
 	// +kubebuilder:scaffold:scheme
 }
 
+var controllers []string
+
+var allControllers = []string{
```

Comment:
Do we need to define allControllers? Adding a controller also requires changing two variables at the same time. Consider simplifying.

Reply:
fixed. `controllers = []string{...}` is enough